src/Core/Migration/V6_7/Migration1743151679AddContextGatewayUrl.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_7;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
```

Comment:
```suggestion
#[Package('checkout')]
```

Reply:
@mitelg the ContextGateway is an app related feature and has nothing to do with the checkout in general, thats why I flagged all of it as framework